draft-ietf-quic-recovery.md

```diff
@@ -467,11 +481,11 @@ kReorderingThreshold (default 3):
   considers a packet lost.
 
 kTimeReorderingFraction (default 1/8):
-: Maximum reordering in time space before time based loss detection considers
+: Maximum reordering in time space before time-based loss detection considers
   a packet lost.  In fraction of an RTT.
 
 kUsingTimeLossDetection (default false):
-: Whether time based loss detection is in use.  If false, uses FACK style
+: Whether time-based loss detection is in use.  If false, uses FACK style
```

Comment:
This is called FACK style here, but probably should be called "packet number based loss detection" for consistency?  It may be worth referencing FACK above when describing packet number based loss detection.

Reply:
I'm not familiar with FACK. Can you suggest something?